### DISCUSSION TITLE
The default trait for Environment.

### DISCUSSION THREAD

**Discussion by cathaysia** [mitsuhiko/minijinja#391] [General]
I noticed that Environment has three create method: 

- [new](https://docs.rs/minijinja/latest/src/minijinja/environment.rs.html#87-103)
- [empty](https://docs.rs/minijinja/latest/src/minijinja/environment.rs.html#109-125)
- [default](https://docs.rs/minijinja/latest/src/minijinja/environment.rs.html#64-66)

the default is actually empty. In general, default should provide reasonable default values. So would it make more sense to point it to new?

**Comment by mitsuhiko** [comment]
I have been thinking about this a few times now and I'm generally not too happy with it having a default at all. It would be a breaking change to change it. Maybe the right thing would be for `default` to match `new` indeed, but not sure what to do now.

**Reply by cathaysia** [reply]
Hi, my friend. I'm very sorry that I forgot about this. I think it is OK whether it is default or new. The key is that I believe that an API should be provided that is unambiguous and meets expectations. As long as this is the case.
